GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -318,7 +365,7 @@ public GVRMesh createQuad(float width, float height) {
      * @return The file as a bitmap, or {@code null} if file path does not exist
      *         or the file can not be decoded into a Bitmap.
      */
-    public Bitmap loadBitmap(String fileName) {
+    public static Bitmap loadBitmap(String fileName) {
```

Comment:
I don't think it's worth making `mContext` static just so that this method becomes `static`. Not only does this generate a warning on every call like `mContext.loadBitmap(whatever)` it makes every reference to `mContext` a bit slower: static variables generally have to be referenced with a full 32 (or 64!) bit address, instead of a 8 or 16 bit offset to a register-resident `this` pointer.